config/services.php

```diff
@@ -35,10 +37,18 @@
                 abstract_arg('path to css output directory'),
                 service('sass.builder'),
             ])
+
         ->set('sass.public_asset_path_resolver', SassPublicPathAssetPathResolver::class)
         ->decorate('asset_mapper.public_assets_path_resolver')
         ->args([
             service('.inner')
         ])
+
+        ->set('sass.listener.pre_assets_compile', PreAssetsCompileEventListener::class)
```

Comment:
The argument is missing: 

```
            ->args([
                service('sass.builder')
            ])
```

Reply:
Sounds valid. Would you like to create a PR?

//cc @WebMamba